src/components/panel/panel.js

```diff
@@ -935,6 +939,11 @@ MdPanelRef.prototype.detach = function() {
     return self._$q.resolve(self);
   };
 
+  if (this._restoreScroll) {
+    this._restoreScroll();
+    delete this._restoreScroll;
```

Comment:
Just set to `null`; it accomplishes the same thing and is generally always faster.